tests/console/rabbitmq.pm

```diff
@@ -20,7 +20,7 @@ use utils 'systemctl';
 
 sub run {
     select_console 'root-console';
-    assert_script_run('zypper -n in rabbitmq-server');
+    assert_script_run('zypper -n in rabbitmq-server wget');
```

Comment:
I recommend to move the installation of wget into the next zypper call to install "python-pika" which we install to execute the test, this line here can be considered part of the test assertions if you get my meaning

Reply:
Done.